<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modbus_master</class>
 <widget class="QWidget" name="modbus_master">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>modbus_master</string>
  </property>
  <widget class="QPushButton" name="ButtonCloseCoil">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭线圈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCoilState">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询线圈状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOpenCoil">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开线圈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonGetGegister">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询开关次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelnumber">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>开关次数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_reg">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelcoil">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>线圈状态：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_coil">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
